<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managemenu</class>
 <widget class="QWidget" name="managemenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Manage Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>811</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/images/managemenubg.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>342</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="priceEdit">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 85, 255);
color: rgb(0, 0, 0);
border-radius:8px;
padding:10px;
font: 300 9pt &quot;IRANYekan&quot;;
border:1px solid black;
width:80px;
height:20px;</string>
        </property>
        <property name="placeholderText">
         <string>قیمت</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeEdit">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 85, 255);
color: rgb(0, 0, 0);
border-radius:8px;
padding:10px;
font: 300 9pt &quot;IRANYekan&quot;;
border:1px solid black;
width:80px;
height:20px;</string>
        </property>
        <property name="locale">
         <locale language="Persian" country="Iran"/>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="placeholderText">
         <string>دسته بندی</string>
        </property>
        <item>
         <property name="text">
          <string>فست فود</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>نوشیدنی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>خورشت</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ایرانی </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>فرنگی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>سنتی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>خانگی </string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 85, 255);
color: rgb(0, 0, 0);
border-radius:8px;
padding:10px;
font: 300 9pt &quot;IRANYekan&quot;;
border:1px solid black;
width:80px;
height:20px;</string>
        </property>
        <property name="placeholderText">
         <string>نام غذا</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="addbtn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:lightgreen;
color: rgb(0, 0, 0);
border-radius:8px;
padding:5px;
font: 300 9pt &quot;IRANYekan&quot;;
border:1px solid black;
width:240px;</string>
      </property>
      <property name="text">
       <string>افزودن غذا به منو</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f77599;
border-radius:8px;</string>
      </property>
      <property name="locale">
       <locale language="Persian" country="Iran"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rokh</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <fontweight>Black</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 900 10pt &quot;Rokh&quot;;</string>
   </property>
   <property name="text">
    <string>تغییر منو</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>121</width>
     <height>38</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
color: rgb(0, 0, 0);
border-radius:8px;
padding:5px;
font: 300 9pt &quot;IRANYekan&quot;;
border:1px solid black;</string>
   </property>
   <property name="text">
    <string>حذف غذا از منو</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(235, 87, 87);
border:0px;
padding:10px;
border-radius:8px;

font: 800 9pt &quot;IRANYekan&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
